felix/fv/wireguard_test.go

```diff
@@ -399,6 +402,23 @@ var _ = infrastructure.DatastoreDescribe("_BPF-SAFE_ WireGuard-Supported", []api
 							}, "30s", "330ms").ShouldNot(BeEmpty())
 						}
 					}
+
```

Comment:
I'm thinking it's probably better to remove changes to `testConfig` struct and move all the changes into a new `It()` section. Then enable NAPI for IPv4 or IPv6 if the testConfig has enabled it. Update the setting and then wait to for eventually to expire. I believe this way the test itself is more self explanatory.

Reply:
Moved to its own It() block.